Gordon360/Services/EmailService.cs

```diff
@@ -57,30 +62,31 @@ public async Task<IEnumerable<EmailViewModel>> GetEmailsForActivityAsync(string
         /// <returns></returns>
         public void SendEmails(string[] to_emails, string from_email, string subject, string email_content, string password)
         {
-            using (var smtp = new SmtpClient())
+            using var smtp = new SmtpClient();
+            var credential = new NetworkCredential
             {
-                var credential = new NetworkCredential
-                {
-                    UserName = from_email,
-                    Password = password
-                };
-                smtp.Credentials = credential;
-                smtp.Host = "smtp.office365.com";
-                smtp.Port = 587;
-                smtp.EnableSsl = true;
-                var message = new MailMessage();
-                message.From = new MailAddress(from_email);
-                message.Bcc.Add(new MailAddress(from_email));
-                foreach (string to_email in to_emails)
-                {
-                    message.To.Add(new MailAddress(to_email));
-                }
-                message.Subject = subject;
-                message.Body = email_content;
-                message.IsBodyHtml = true;
+                UserName = from_email,
+                Password = password
+            };
+            smtp.Credentials = credential;
+            smtp.Host = "smtp.office365.com";
+            smtp.Port = 587;
+            smtp.EnableSsl = true;
```

Comment:
I wasn't really refactoring this code. That change was automated by the linter because VS made a formatting suggestion.

In reality, this is actually getting email and password from the body of the send email request. Currently, it's completely unused because obviously that's a terrible way to integrate with email in 2023.